assets/wizards/site-design/index.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * Site Design
+ */
+
+/**
+ * WordPress dependencies.
+ */
+import { Component, render, Fragment } from '@wordpress/element';
+import apiFetch from '@wordpress/api-fetch';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Material UI dependencies.
+ */
+import HeaderIcon from '@material-ui/icons/Web';
+
+/**
+ * Internal dependencies.
+ */
+import { withWizard } from '../../components/src';
+import Router from '../../components/src/proxied-imports/router'
+import { ThemeSelection } from './views';
+
+const { HashRouter, Redirect, Route, Switch } = Router;
+
+class SiteDesignWizard extends Component {
+
+	/**
+	 * Constructor.
+	 */
+	constructor() {
```

Comment:
When the constructor does nothing but initialise state, you can remove the constructor and make state a class property, e.g.:

```
class SiteDesignWizard extends Component {
	state = {
		theme: null,
	};
```

Reply:
Done in https://github.com/Automattic/newspack-plugin/pull/421/commits/a9bd0dd7762bc6dbe0e2b1ca57139bae13548036